src/rpc/service_eth.go

```diff
@@ -95,6 +95,7 @@ func (s *EthService) NewTimeEvent(ctx context.Context, newTime int64) {
 				txHash, err := s.SendRawTransaction(ctx, rawTx)
 
 				if err != nil {
+					metrics.MetricsErrorReturnedCounter.Desc()
```

Comment:
This doesn't decrement the counter, it's the counter descriptor. I think you can add another counter here for internal errors, and we can later subtract one from the other when we want to visualize them, that way we also have both types of error counts.